From ba0b36d910131e7d9e804822b29692a737b9a714 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 9 Sep 2020 08:46:34 +0200
Subject: [PATCH 1587/1852] dt-bindings: soc: xilinx: ai-engine: Add 64bit bus
 to example

commit 491e2d7111c21bf220446cfa98587dc747bace95 from
https://github.com/Xilinx/linux-xlnx.git

Current example in binding is missing bus description. By default 32bit
addresses/sizes are expected. Example is using 64bit instead that's why it
is necessary to fix it.

The following error is generated when dt_binding_check runs:
.../xlnx,ai-engine.example.dt.yaml: example-0: ai-engine@20000000000:reg:0:
[512, 0, 1, 0] is too long

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Reviewed-by: Wendy Liang <wendy.liang@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../bindings/soc/xilinx/xlnx,ai-engine.yaml   | 33 +++++++++++--------
 1 file changed, 19 insertions(+), 14 deletions(-)

diff --git a/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai-engine.yaml b/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai-engine.yaml
index 25bbede01608..ec6239c1eb59 100644
--- a/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai-engine.yaml
+++ b/Documentation/devicetree/bindings/soc/xilinx/xlnx,ai-engine.yaml
@@ -94,22 +94,27 @@ patternProperties:
 examples:
   - |
     #include <dt-bindings/power/xlnx-versal-power.h>
-    ai_engine: ai-engine@20000000000 {
-      compatible = "xlnx,ai-engine-v1.0";
-      reg = <0x200 0x0 0x1 0x0>;
+    bus {
       #address-cells = <2>;
       #size-cells = <2>;
-      power-domains = <&versal_firmware PM_DEV_AI>;
-      interrupt-parent = <&gic>;
-      interrupts = <0x0 0x94 0x4>,
-                   <0x0 0x95 0x4>,
-                   <0x0 0x96 0x4>;
-      interrupt-names = "interrupt1", "interrupt2", "interrupt3";
-
-      aie_partition0: aie-partition@0 {
-        /* 50 columns and 8 core tile rows + 1 SHIM row */
-        reg = <0 0 50 9>;
-        xlnx,partition-id = <1>;
+
+      ai_engine: ai-engine@20000000000 {
+        compatible = "xlnx,ai-engine-v1.0";
+        reg = <0x200 0x0 0x1 0x0>;
+        #address-cells = <2>;
+        #size-cells = <2>;
+        power-domains = <&versal_firmware PM_DEV_AI>;
+        interrupt-parent = <&gic>;
+        interrupts = <0x0 0x94 0x4>,
+                        <0x0 0x95 0x4>,
+                        <0x0 0x96 0x4>;
+        interrupt-names = "interrupt1", "interrupt2", "interrupt3";
+
+        aie_partition0: aie-partition@0 {
+                /* 50 columns and 8 core tile rows + 1 SHIM row */
+                reg = <0 0 50 9>;
+                xlnx,partition-id = <1>;
+        };
       };
     };
 
-- 
2.31.1

